lib/common/digest.c

```diff
@@ -9,11 +9,15 @@
 
 #include <crm_internal.h>
 
+#include <stdbool.h>
 #include <stdio.h>
 #include <unistd.h>
 #include <string.h>
 #include <stdlib.h>
+
+#include <glib.h>               // GString, etc.
 #include <gnutls/crypto.h>
+#include <md5.h>
```

Comment:
f5b6b67c8b1d019e6c34975c9afb84823308e737 got rid of this include line, and it being back is what's causing the CI failures.